relay-statsd/src/lib.rs

```diff
@@ -17,13 +17,12 @@
 //! or the [`metric!`] macro will become a noop. Only when configured, metrics will actually be
 //! collected.
 //!
-//! To initialize the client, either use [`set_client`] to pass a custom client, or use
-//! [`init`] to create a default client with known arguments:
+//! To initialize the client use [`init`] to create a default client with known arguments:
```

Comment:
I left the original doc comment intact, but the rest of this file is a modified copy of https://github.com/getsentry/symbolicator/blob/1873ebad53b96edb8ba1380e3d9cff824ae3de86/crates/symbolicator-service/src/metrics.rs.